tests/execution_engine/test_pandas_execution_engine.py

```diff
@@ -53,14 +53,93 @@ def test_reader_fn():
     assert "<function" in str(fn_new)
 
 
+def test_get_full_access_compute_domain_with_column_domain():
+    engine = PandasExecutionEngine()
+    df = pd.DataFrame(
+        {"a": [1, 2, 3, 4, 5], "b": [2, 3, 4, 5, None], "c": [1, 2, 3, 4, None]}
+    )
+    expected_column_df = df.iloc[:3]
+
+    # Loading batch data
+    engine.load_batch_data(batch_data=df, batch_id="1234")
+    data = engine.get_full_access_compute_domain(
+        domain_kwargs={
+            "column": "a",
+            "row_condition": "b<5",
+            "condition_parser": "pandas",
+        }
+    )
+    assert data.equals(
+        expected_column_df
+    ), "Data does not match after getting full access compute domain"
+
```

Comment:
Does this `data.equals()` call return "Data does not match" because we are selecting column a, but row_condition refers to column b? If so, could we add a little note saying why the test is expected to return "Data does not match?"

Reply:
@Shinnnyshinshin Because this is regarding the way `assert` works, it is easier to answer it here: we are asserting that the data *will* equal.  In case it does not, we say "sorry, does not match".  Hope this makes it a little clearer.

Reply:
Ah my mistake :) Thanks for the clarification